06week/Checkpoint2

```diff
@@ -0,0 +1,126 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+
+const nums =[];
+for (let i in strNums) {
+  const n = parseInt(strNums[i]);
+  nums.push(n);
+}
+console.log(nums)
+
+
+// Find the sum of the even values
+
+let sumEvens=0;
+nums.forEach(num =>{
```

Comment:
forEach returns undefined.  While this works, you should consider using other methods to return the expected values.  Also, the result of the written function should be assigned to a constant variable when possible.